packages/core/mesh/edge-client/src/client.ts

```diff
@@ -76,6 +77,17 @@ export class EdgeClient implements EdgeConnection {
     return !!this._ws;
   }
 
+  setIdentity({ deviceKey, identityKey }: { deviceKey: PublicKey; identityKey: PublicKey }) {
+    // TODO(mykola): Make synchronous. Manual close should cancel the operation.
+    this._deviceKey = deviceKey;
+    this._identityKey = identityKey;
+    this.close()
+      .then(async () => {
+        await this.open();
+      })
+      .catch(log.catch);
```

Comment:
```suggestion
      .catch(err => log.catch(err));
```

passing log functions like this prevents the log metadata from being inserted (filenames, line numbers)